lib/src/Uncertainty/Distribution/StudentFunctions.cxx

```diff
@@ -49,6 +49,9 @@ Scalar StudentCDF(const Scalar nu,
 {
   if (!(nu > 0.0)) throw InvalidArgumentException(HERE) << "Error: nu must be positive, here nu=" << nu;
   if (x == 0.0) return 0.5;
+#ifdef OPENTURNS_HAVE_BOOST
+  return (tail ? boost::math::cdf(boost::math::complement(boost::math::students_t(nu), x)) : boost::math::cdf(boost::math::students_t(nu), x)) ;
+#else
```

Comment:
If Boost is not available, we still have the bug. The problem is that the formulas are correct but should be rewritten for large |x| by dividing both the numerator and the denominator by their leading terms $|x|^{\nu/2}$. Be careful, the sign of x will show up in the numerator, but as the numerator is an odd polynomial function, you just have to select the same formula when (tail != (x<0.0)). For example, when $\nu=2$, we should have:
```
if (nu == 2.0)
{
  if (std::abs(x) < 1.0) return (tail ? 0.5 - 0.5 * (x / std::sqrt(2.0 + x2)) : 0.5 + 0.5 * (x / std::sqrt(2.0 + x2)));
  // tail xor (x < 0.0)
  return (tail != (x < 0.0) ? 0.5 - 0.5 / std::sqrt(2.0 / x2 + 1.0) : 0.5 + 0.5 / std::sqrt(2.0 / x2 + 1.0));
}
```

Reply:
thanks for the hint, I rewrote all the formulas